pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -58,6 +58,15 @@ const (
 	warningTime = 40 * time.Second
 	// Maximum allowed number of flow messages enqueued and waiting to be published.
 	flowlogQueueCap = 100
+
+	// Factor by which zedagentContext.dormantTimeScaleFactor must be scaled up/down,
+	// depending upon metrics publish result.
+	metricsFailureScaleFactor = 2
+
+	// Upper limit of zedagentContext.dormantTimeScaleFactor.
+	maxDormantTimeScaleFactor = 8
+	// Lower limit of zedagentContext.dormantTimeScaleFactor.
+	minDormantTimeScaleFactor = 1
```

Comment:
AFAICT this should still be 3 as we have hard-coded today.

Reply:
The reason for that is that otherwise a single failure would result in marking it as suspect since the factor would be 1 until the first failure.

Reply:
Changed to 3